### DISCUSSION TITLE
I've made a fork that publishes releases in GitHub releases.

### DISCUSSION THREAD

**Discussion by ghost** [atom-ide-community/atom#431] [Show and tell]
It's here: [https://github.com/stechness/atomcommunity-github-releases-windows/releases/tag/1.63.0-dev](https://github.com/stechness/atomcommunity-github-releases-windows/releases/tag/1.63.0-dev)
**FOR THOSE VIEWING NOW: That project will be DISCONTINUED and REPLACED by @autumnblazey 's dashboard.**

**Comment by aminya** [comment]
Awesome. Do you want to make a PR that does it for this repository?

**Reply by ghost** [reply]
Yeah, that's better than having to do more CI work, easier that way.

**Reply by ghost** [reply]
However, I'm not fully sure how to do this, since my fork is not writing code differences however, so I don't know. Maybe just transfer the binaries and download them, make a release here?

**Reply by aminya** [reply]
Oh, OK. I thought it is automated. Yeah, we can release them manually, but it is easier to automate this. I am pretty sure Azure provides this.

**Reply by ghost** [reply]
@aminya Any updates on this?

**Reply by ghost** [reply]
oh, I just started a thread on discord here: https://discord.com/channels/737254137690980372/994328685446451241
